plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -123,19 +119,15 @@ def temp_kinit(principal, password):
 
 
 def temp_kdestroy(ccache_dir, ccache_name):
-    """
-    Destroy temporary ticket and remove temporary ccache
-    """
+    """Destroy temporary ticket and remove temporary ccache."""
     if ccache_name is not None:
         run([paths.KDESTROY, '-c', ccache_name], raiseonerr=False)
     if ccache_dir is not None:
         shutil.rmtree(ccache_dir, ignore_errors=True)
 
 
 def api_connect(context=None):
-    """
-    Create environment, initialize api and connect to ldap2
-    """
+    """Create environment, initialize api and connect to ldap2."""
```

Comment:
I'd rephrase it to:
```python
"""
Initialize IPA API with the provided context.

`context` can be any of  `server`, `ansible-freeipa` and `cli_installer`, default to `server`.
"""
```

BTW, why isn't `server` the default value instead of `None`?

Reply:
I don't actually know that answer :)
Maybe we should open an issue for that?